src/js/components/auditlogs/eventdetails/terminalplayer.js

```diff
@@ -102,7 +103,7 @@ const generateHtml = (versions, content) => {
           }
           const item = content[contentIndex];
           contentIndex += 1;
-          let delay = 1;
+          let delay = 1000;
```

Comment:
that will result in all messages getting at least 1000ms delayed, I use the `let delay = 1` only to call all messages with the same `setTimeout(processContent, delay)` and only override the delay when needed..

Reply:
that was me tyring to debug. thanks a lot for taking a time to look at it.